src/rotation-lock.cpp

```diff
@@ -77,7 +77,7 @@ class RotationLockIndicator::Impl
 
     if (!ayatana_common_utils_is_lomiri())
     {
-        rotation_lock_icon_name = "display";
+        rotation_lock_icon_name = "display-panel";
```

Comment:
I can't find an icon of that name on my system. Where do you think display-panel(.png|.svg) should come from?

Reply:
It will be up to the theme to provide *-panel icons in the future, just as some themes already do for the Sound Indicator. If no such icons are present, the indicator uses the next available icon, so there is no visible change.